Tests/test_align.py

```diff
@@ -176,6 +176,8 @@ def test_basic_alignment(self):
         alignment.append(SeqRecord(Seq(letters), id="mixed"))
         alignment.append(SeqRecord(Seq(letters.lower()), id="lower"))
         alignment.append(SeqRecord(Seq(letters.upper()), id="upper"))
+        alignment.append(SeqRecord(Seq(letters), id="duplicate"))
+        del alignment[3]
```

Comment:
Maybe confirm the number of rows in the alignment after this?

Reply:
I thought of it but didn't do it because the test already checks the number of rows and columns after the delete:
`181 self.assertEqual(alignment.get_alignment_length(), 26)` column check
`182 self.assertEqual(len(alignment), 3)`                                      row check

It does this to check for the appends before too, if its better I could put the test of the **del** method after it does those tests and check for the row length again just in case.

Reply:
Ah, that's fine then.